pages/dashboard.js

```diff
@@ -13,46 +13,79 @@ const Content = styled.div`
   width: 100%;
 `;
 
-const getProfile = (data) => {
-  return data ? data.accountByEmail.data : [];
-}
-
 export default function Dashboard() {
   const user = useUser({ redirectTo: '/signup', redirectIfFound: false });
-  const { data, errorMessage } = getProfileByEmail('khobra.z@gmail.com');
-  const [ profile, setProfile ] = useState([]);
+  const [profile, setProfile] = useState(null);
+  const [email, setEmail] = useState(null);
+  const [applications, setApplications] = useState([]);
+  const [errorMessage, setErrorMessage] = useState(null);
   const { state } = useContext(Context);
 
-  const name = 'Zainab Ebrahimi';
-  const email = 'khobra.z@gmail.com';
-  const company = 'Tall Poppy';
-  const interviewDate = 'June 11 2021 @ 12pm ET';
-  const interviewType = 'Technical Interview';
-  const role = 'Software Engineer';
-  const excitement = '2';
-  const nerves = '2';
-
   useEffect(() => {
-    if (!profile.length) {
-      setProfile(getProfile(data))
+    async function fetchAccount() {
```

Comment:
do we need to define a function here, given you're calling it right after definition?

Reply:
yep, react forces you to define your useEffect as a standalone async function (then call it right after) if you're fetching data asynchronously. see [here](https://stackoverflow.com/questions/53332321/react-hook-warnings-for-async-function-in-useeffect-useeffect-function-must-ret)

Reply:
oh interesting! good to know. 👍🏼